awx/playbooks/project_update.yml

```diff
@@ -237,10 +239,12 @@
           vars:
             req_file: "{{ lookup('ansible.builtin.first_found', req_candidates, skip=True) }}"
             req_candidates:
-              - "{{ project_path | quote }}/collections/requirements.yml"
-              - "{{ project_path | quote }}/collections/requirements.yaml"
-              - "{{ project_path | quote }}/requirements.yml"
-              - "{{ project_path | quote }}/requirements.yaml"
+              files:
+                - "{{ project_path | quote }}/collections/requirements.yml"
+                - "{{ project_path | quote }}/collections/requirements.yaml"
+                - "{{ project_path | quote }}/requirements.yml"
+                - "{{ project_path | quote }}/requirements.yaml"
```

Comment:
The inclusion of these 2 `requirements` files in the project root seems to be failing some tests.  These extra paths were added in https://github.com/ansible/awx/commit/4286d411a74024b634cefa49b0c3cbdcd71a7231

I'm trying to look through docs now, to see what the description of a project root `requirements.yml` is intended to be.  For the record we effectively have a v1 and v2 format for `requirements.yml`, where `v1` is just roles, and `v2` can contain collections and roles.  The failures make it appear as though a project root may have just been `v1`.